Documentation/WhenToUseBabylonNative.md

```diff
@@ -0,0 +1,194 @@
+-   [My app's platform doesn't inherently use JavaScript.](#do-you-have-exceptional-rendering-requirements-such-as-aaa-quality-graphics-or-extreme-performance-needs)
+
+***
+
+## Babylon Native is probably a good choice for your project!
+
+Congratulations! If you ended up here, then we think Babylon Native may
+be a suitable choice for adding 3D to your project. Typically, 
+applications for which Babylon Native is well-suited tend to be developed
+as true native apps (C++, C#, etc.) which require 3D rendering as a 
+feature, though not necessarily the focus. Examples might include the
+following:
+
+-   A presentation app written in C# that needs to be able to render 3D 
+    as well as 2D content.
+-   A scientific simulation utility written in Java that wants to be able 
+    to render data in 3D.
+-   A 3D content generation tool written in C++ that wants to be able to 
+    preview its output.
+
+The main reason Babylon Native is well-suited for these scenarios is 
+because Babylon Native makes it possible to bring Babylon.js's 
+rendering capabilities into native applications with *extremely* minimal
+requirements regarding app and development infrastructure. In other words,
+Babylon Native is perhaps the simplest way to add Web-style 3D rendering
+to a native application without having to pivot that application to 
+*become* a Web app.
+
+***
+
+## Babylon React Native is probably a good choice for your project!
+
+Congratulations! If you ended up here, then we think Babylon React Native
+may be a suitable choice for adding 3D to your project. There are two main
+situations when Babylon React Native is a particularly good choice for a 
+project:
+
+-   When the project is already being built with React Native and wants to
+    add 3D as simply as possible.
+-   When the project requires capabilities that aren't typically exposed
+    to Web-like platforms.
+
+In the former case, if your app is already committed to React Native as a 
+platform, Babylon React Native is one of two options to add 3D to
+your project using Babylon.js. The second of these options is to simply
+use Babylon.js in a WebView, and there are some advantages to this 
+approach: it can sometimes yield better JavaScript performance (on iOS,
+WebViews are allowed to run JavaScript with JIT enabled while React 
+Native, and consequently Babylon React Native, are not) and it will 
+provide access to the very latest Babylon.js features, which may not 
+always be available in Babylon React Native. However, there are 
+disadvantages as well: a WebView will create a second JavaScript context
+distinct from React Native's main JavaScript context and you will have to
+manage communication between the contexts manually, which can be complex
+and may negatively impact memory footprint as you pay the memory cost for
+two JavaScript contexts instead of one. Furthermore, running inside a 
```

Comment:
It may also be prohibitively slow to send the data across.

Reply:
I'd assume that would depend on how chatty the relationship is, so I don't know how broad a claim we can make, but I'll add a callout to potential performance impacts.